valhalla/jawn/src/controllers/public/playgroundController.ts

```diff
@@ -127,6 +127,12 @@ export class PlaygroundController extends Controller {
         try {
           const response = await openai.chat.completions.create(
             {
+              provider: isOnPrem
+                ? undefined
+                : {
+                    sort: "throughput",
+                    order: ["Fireworks"],
+                  },
```

Comment:
```suggestion
              provider: isOnPrem
                ? undefined
                : {
                    sort: "throughput",
                    order: ["Fireworks"],
                  },
```
let's get rid of this - again probably just merged in but not needed